content/docs/examples/staticsite.md

```diff
@@ -35,11 +40,11 @@ WSL instructions can be found here: **[Install Node.js on Windows Subsystem for
 The initial file structure should resemble the image below. If this is not your starting point for this example, consult the **[Getting Started](/docs/gettingstarted)** section to initialize a `Cdev project` before continuing.
```

Comment:
Add the `cdev init` command so that the user can copy and paste it to start a new project.